pygccxml/parser/scanner.py

```diff
@@ -154,6 +154,11 @@ def __init__(self, xml_file, decl_factory, config, *args):
         self.__mangled_suffix = ' *INTERNAL* '
         self.__mangled_suffix_len = len(self.__mangled_suffix)
 
+        self.__name_attrs_to_skip = []
+        self.__read_location = \
+            lambda decl, attrs, to_skip: self.__read_location_bootstrap(
```

Comment:
This change improves binding generation time by about a minute for me, but I'm not all that happy with the monkey patching solution. If you can think of a different way to avoid the check, I can test that.